drivers/vulkan/vulkan_context.cpp

```diff
@@ -740,7 +740,7 @@ Error VulkanContext::_create_physical_device() {
 		{ 0x10DE, "NVIDIA" },
 		{ 0x13B5, "ARM" },
 		{ 0x5143, "Qualcomm" },
-		{ 0x8086, "INTEL" },
+		{ 0x8086, "Intel" },
```

Comment:
```suggestion
		{ 0x8086, "Intel" },
		{ 0x106B, "Apple" },
```

Reply:
I added it above – I sorted the list from lowest to highest ID.